pkgs/tools/misc/topgrade/default.nix

```diff
@@ -13,11 +13,15 @@ rustPlatform.buildRustPackage rec {
 
   cargoSha256 = "1y85hl7xl60vsj3ivm6pyd6bvk39wqg25bqxfx00r9myha94iqmd";
 
+  buildInputs = stdenv.lib.optionals stdenv.isDarwin [
```

Comment:
Nit: an we use `lib.optional` to make this a bit shorter?

```diff
-  buildInputs = stdenv.lib.optionals stdenv.isDarwin [
-    Foundation
-  ];
+  buildInputs = stdenv.lib.optional stdenv.isDarwin Foundation;
```

Reply:
I personally prefer the later as it minimizes future diffs, but I have made this change.